### ISSUE TITLE
Enter the Coliseum: max() arg is an empty sequence

### ISSUE THREAD

**Issue by beheh** [jleclanche/fireplace#292]
```
ValueError: max() arg is an empty sequence
```

[Full log](https://gist.githubusercontent.com/beheh/d82844936145d86c99b7/raw/ef2fcd03f1abf6893b3b65fec213ee839949d728/gistfile1.txt).

```
Traceback (most recent call last):
  File "./test_fuzzy.py", line 71, in main
    card.play(target=target, choose=choice)
  File "/home/benedict/Dokumente/fireplace/fireplace/card.py", line 280, in play
    self.game.queue_actions(self.controller, [actions.Play(self, target, index, choose)])
  File "/home/benedict/Dokumente/fireplace/fireplace/game.py", line 142, in queue_actions
    ret = self.trigger_actions(source, actions)
  File "/home/benedict/Dokumente/fireplace/fireplace/game.py", line 166, in trigger_actions
    ret.append(action.trigger(source))
  File "/home/benedict/Dokumente/fireplace/fireplace/actions.py", line 148, in trigger
    self.do(source, *args)
  File "/home/benedict/Dokumente/fireplace/fireplace/actions.py", line 360, in do
    source.game.queue_actions(card, [Battlecry(battlecry_card, card.target)])
  File "/home/benedict/Dokumente/fireplace/fireplace/game.py", line 142, in queue_actions
    ret = self.trigger_actions(source, actions)
  File "/home/benedict/Dokumente/fireplace/fireplace/game.py", line 166, in trigger_actions
    ret.append(action.trigger(source))
  File "/home/benedict/Dokumente/fireplace/fireplace/actions.py", line 493, in trigger
    ret.append(self.do(source, target, *target_args))
  File "/home/benedict/Dokumente/fireplace/fireplace/actions.py", line 625, in do
    source.game.queue_actions(source, actions)
  File "/home/benedict/Dokumente/fireplace/fireplace/game.py", line 142, in queue_actions
    ret = self.trigger_actions(source, actions)
  File "/home/benedict/Dokumente/fireplace/fireplace/game.py", line 166, in trigger_actions
    ret.append(action.trigger(source))
  File "/home/benedict/Dokumente/fireplace/fireplace/actions.py", line 487, in trigger
    targets = self.get_targets(source, args[0])
  File "/home/benedict/Dokumente/fireplace/fireplace/actions.py", line 462, in get_targets
    ret = t.eval(source.game, source)
  File "/home/benedict/Dokumente/fireplace/fireplace/dsl/selector.py", line 114, in eval
    merge_output += op.merge(self, merge_input)
  File "/home/benedict/Dokumente/fireplace/fireplace/dsl/selector.py", line 314, in merge
    highest = self.func(entities, key=key).tags.get(self.tag, 0)
```

**Comment by smallnamespace** [comment]
I'll write a test and make sure the new selectors handle this

**Comment by jleclanche** [comment]
Fixed by #317
